client/src/cordova/plugin/apple/src/OutlinePlugin.swift

```diff
@@ -154,10 +154,12 @@ class OutlinePlugin: CDVPlugin {
     DDLogInfo("Invoking Method \(methodName) with input \(input)")
     Task {
       guard let result = OutlineInvokeMethod(methodName, input) else {
+        DDLogInfo("InvokeMethod \(methodName) got nil result")
```

Comment:
Error instead of Info?
```suggestion
        DDLogError("InvokeMethod \(methodName) got nil result")
```

Reply:
Updated